### DISCUSSION TITLE
Welcome to phd-cs-us Discussions!

### DISCUSSION THREAD

**Discussion by nguyenthanhvuh** [nguyenthanhvuh/phd-cs-us#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by anhtienng** [comment]
Hi Vu,
I'm Anh Tien Nguyen, a Master's student at Korea University. I am interested in applying deep learning for medical images.
I'm preparing a CS PhD 2025 admission and found your document is amazing. 
Thank you for sharing.

**Reply by nguyenthanhvuh** [reply]
you're welcome,  glad you find it useful.  How did you know about the document?

**Reply by anhtienng** [reply]
I found your sharing on VietPhD.

**Comment by lam-thai-nguyen** [comment]
Hi Prof. Nguyen and everyone,

My name is Lam Thai Nguyen, and you can call me Thai.

I am currently in my third year in **Control Engineering and Automation** at **VNU-UET, Vietnam**. I developed an interest in research in AI a year ago. I have been working in a research group at my university, and we are currently focusing on **using transfer learning on crack segmentation models to solve the structural analysis challenge in rice panicles**.

I am planning to apply for a **CS PhD in the US** upon my graduation. As a Vietnamese non-CS undergraduate, I face significant competition. My GPA is 3.35, which is sufficient but not exceptional. My research experience is limited, and my research output reflects my current ability in this field (i.e., a beginner). Additionally, my advisor and research group are not well-known to US admissions committees or professors.

I discovered Prof. Nguyen's repository and documentation while feeling confused and skeptical about my chances of being accepted. I read all 61 pages in one go and found the insights extremely helpful.

I wanted to share my story and express my gratitude to Prof. Nguyen. Your work has provided clarity and guidance for me. I hope other PhD applicants like me, with limited advantages, can also find and appreciate your work.

Thank you for your efforts,

Thai

**Reply by nguyenthanhvuh** [reply]
You're welcome!  The document is intended exactly to help students like yourself. Good luck with your application when you graduate.
